### ISSUE TITLE
Query parameters for site search

### ISSUE THREAD

**Issue by zombor** [MatteoGabriele/vue-analytics#31]
Hi, I'm trying to use site search on GA, but it doesn't appear to be getting any data. Any chance that query params aren't being sent to GA? I'm not exactly sure if it gets it from the `page` or `location` key. `page` doesn't seem like it would include it.

**Comment by MatteoGabriele** [comment]
Hi @zombor 
Could you please provide some code you are using? Like which options are u passing to the plugin, which method doesn't work,  how you are implementing that and which version of vue-analytics

Thanks :)

**Comment by zombor** [comment]
Yep, I should have done that in the first place! It's all pretty standard, just using auto-page tracking:

```javascript
if (process.env.NODE_ENV === 'production') {
  const gaId = 'UA-xxxxxxxx-1';
  Vue.use(VueAnalytics, { router, id: gaId });
}
```

My package.json has `"vue-analytics": "^3.0.1"`.

**Comment by MatteoGabriele** [comment]
hey @zombor thanks but I still don't understand what you need to track, what's the expected behavior you're looking for

thanks

**Comment by zombor** [comment]
I'm trying to use the Site Search feature in GA, but it's not getting any data showing up in GA with the query param that I use for my search page. I have a hunch that it needs the query params sent in the `page` param of `pageview`, but this package only sends the path: https://github.com/MatteoGabriele/vue-analytics/blob/master/src/features/page.js#L11

`location` *does* send the full url with query, but I don't know for sure if that's what GA is expecting.

**Comment by MatteoGabriele** [comment]
ok I'll have a look! thanks! :)

**Comment by MatteoGabriele** [comment]
@zombor the feature is available on `v4.0.2`
